Applications/StableDiffusionExample/ContentView.swift

```diff
@@ -0,0 +1,310 @@
+import SwiftUI
+
+struct ContentView: View {
+
+    @State var prompt = "dismal swamp, dense, very dark, realistic"
+    @State var negativePrompt = ""
+    @State var evaluator = StableDiffusionEvaluator()
+    @State var showProgress = false
+
+    var body: some View {
+        VStack {
+            HStack {
+                if let progress = evaluator.progress {
+                    ProgressView(progress.title, value: progress.current, total: progress.limit)
+                }
+            }
+            .frame(height: 20)
+
+            Spacer()
+            if let image = evaluator.image {
+                Image(image, scale: 1.0, label: Text(""))
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(minHeight: 200)
+            }
+            Spacer()
+
+            Grid {
+                GridRow {
+                    TextField("prompt", text: $prompt)
+                        .onSubmit(generate)
+                        .disabled(evaluator.progress != nil)
+                        #if os(visionOS)
+                            .textFieldStyle(.roundedBorder)
+                        #endif
+
+                    Button(action: { prompt = "" }) {
+                        Label("clear", systemImage: "xmark.circle.fill").font(.system(size: 10))
+                    }
+                    .labelStyle(.iconOnly)
+                    .buttonStyle(.plain)
+
+                    Button("generate", action: generate)
+                        .disabled(evaluator.progress != nil)
+                        .keyboardShortcut("r")
+                }
+                if evaluator.modelFactory.canShowProgress
+                    || evaluator.modelFactory.canUseNegativeText
+                {
+                    GridRow {
+                        if evaluator.modelFactory.canUseNegativeText {
+                            TextField("negative prompt", text: $negativePrompt)
+                                .onSubmit(generate)
+                                .disabled(evaluator.progress != nil)
+                                #if os(visionOS)
+                                    .textFieldStyle(.roundedBorder)
+                                #endif
+                            Button(action: { prompt = "" }) {
+                                Label("clear", systemImage: "xmark.circle.fill").font(
+                                    .system(size: 10))
+                            }
+                            .labelStyle(.iconOnly)
+                            .buttonStyle(.plain)
+                        } else {
+                            EmptyView()
+                            EmptyView()
+                        }
+
+                        if evaluator.modelFactory.canShowProgress {
+                            Toggle("Show Progress", isOn: $showProgress)
+                        }
+                    }
+                }
+            }
+            .frame(minWidth: 300)
```

Comment:
The UI:

<img width="654" alt="image" src="https://github.com/user-attachments/assets/1f5e7416-c97d-4451-b7ef-b12650dc3ba3">

A progress bar at the top, image in the middle, prompt at the bottom.